[esx_addons]/esx_mechanicjob/client/main.lua

```diff
@@ -822,28 +818,31 @@ end)
 -- Display markers
 CreateThread(function()
 	while true do
-		local Sleep = 2000
+		Wait(0)
 
 		if ESX.PlayerData.job and ESX.PlayerData.job.name == 'mechanic' then
-			Sleep = 500
 			local coords, letSleep = GetEntityCoords(PlayerPedId()), true
 
 			for k,v in pairs(Config.Zones) do
 				if v.Type ~= -1 and #(coords - v.Pos) < Config.DrawDistance then
-					Sleep = 0
 					DrawMarker(v.Type, v.Pos.x, v.Pos.y, v.Pos.z, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, v.Size.x, v.Size.y, v.Size.z, v.Color.r, v.Color.g, v.Color.b, 100, false, true, 2, false, nil, nil, false)
 					letSleep = false
 				end
 			end
+
+			if letSleep then
+				Wait(500)
+			end
```

Comment:
Code like this it's just wrong... please don't use that. Only change the value when needed.